<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Ph. Faist</author>
 <class>KLFLatexSymbols</class>
 <widget class="QWidget" name="KLFLatexSymbols">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KLatexFormula - LaTeX symbols</string>
  </property>
  <property name="windowIcon">
   <iconset resource="klfres.qrc">
    <normaloff>:/pics/klatexformula-128.png</normaloff>:/pics/klatexformula-128.png</iconset>
  </property>
  <layout class="QGridLayout" name="lyt_KLFLatexSymbols">
   <property name="margin">
    <number>5</number>
   </property>
   <item row="1" column="0">
    <widget class="QPushButton" name="btnClose">
     <property name="toolTip">
      <string>Close this dialog</string>
     </property>
     <property name="text">
      <string>&amp;Close</string>
     </property>
     <property name="icon">
      <iconset resource="klftoolsres.qrc">
       <normaloff>:/pics/closehide.png</normaloff>:/pics/closehide.png</iconset>
     </property>
     <property name="shortcut">
      <string>Alt+C</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QFrame" name="frmMain">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QGridLayout" name="lyt_frmMain">
      <property name="horizontalSpacing">
       <number>4</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="0" colspan="3">
       <widget class="QFrame" name="frmStackContainer">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QComboBox" name="cbxCategory">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Choose symbol category</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="KLFSearchBar" name="searchbar">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>10</height>
         </size>
        </property>
        <property name="autoHide" stdset="0">
         <bool>true</bool>
        </property>
        <property name="focusOutText" stdset="0">
         <string>Search for Symbols ...</string>
        </property>
        <property name="showSearchLabel" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QToolButton" name="btnSearch">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>&lt;p style=&quot;white-space: pre&quot;&gt;Search for Symbols ...  [&lt;strong&gt;Ctrl-F&lt;/strong&gt;, &lt;strong&gt;Ctrl-S&lt;/strong&gt; or &lt;strong&gt;/&lt;/strong&gt;]</string>
        </property>
        <property name="icon">
         <iconset resource="klfres.qrc">
          <normaloff>:/pics/showpreview.png</normaloff>:/pics/showpreview.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QToolButton" name="btnInsertSearchCurrent">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>&lt;p style=&quot;white-space: pre&quot;&gt;Insert the current search result match  [&lt;strong&gt;Ctrl-Space&lt;/strong&gt;]&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="klfres.qrc">
          <normaloff>:/pics/insertsymb.png</normaloff>:/pics/insertsymb.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ctrl+Space</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="2" margin="0"/>
 <customwidgets>
  <customwidget>
   <class>KLFSearchBar</class>
   <extends>QFrame</extends>
   <header location="global">klfsearchbar.h</header>
   <container>1</container>
   <slots>
    <signal>visibilityChanged(bool)</signal>
    <signal>searchPerformed(bool)</signal>
    <signal>hasMatch(bool)</signal>
    <slot>focus()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cbxCategory</tabstop>
  <tabstop>btnClose</tabstop>
 </tabstops>
 <resources>
  <include location="klfres.qrc"/>
  <include location="klftoolsres.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnSearch</sender>
   <signal>toggled(bool)</signal>
   <receiver>searchbar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>494</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>460</x>
     <y>461</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchbar</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>btnSearch</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>494</x>
     <y>425</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSearch</sender>
   <signal>pressed()</signal>
   <receiver>searchbar</receiver>
   <slot>focus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>494</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>458</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchbar</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>btnInsertSearchCurrent</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>434</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>473</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchbar</sender>
   <signal>hasMatch(bool)</signal>
   <receiver>btnInsertSearchCurrent</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>447</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>444</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
